src/SourceBuild/content/eng/tools/tasks/Microsoft.DotNet.SourceBuild.Tasks.LeakDetection/CheckForPoison.cs

```diff
@@ -375,8 +386,9 @@ private static PoisonedFileEntry ExtractAndCheckZipFileOnly(IEnumerable<CatalogP
 
             foreach (var child in Directory.EnumerateFiles(tempDir, "*", SearchOption.AllDirectories))
             {
-                // also add anything in this zip/package for checking
-                futureFilesToCheck.Enqueue(child);
+                string DisplayPath = $"{candidate.DisplayPath}/{child.Replace(tempDir, string.Empty).TrimStart(Path.DirectorySeparatorChar)}";
```

Comment:
```suggestion
                string displayPath = $"{candidate.DisplayPath}/{child.Replace(tempDir, string.Empty).TrimStart(Path.DirectorySeparatorChar)}";
```

Reply:
Updated in this commit: https://github.com/dotnet/installer/pull/17438/commits/11232daadf922b0592118dd9f588e6e77854f66b